src/Utilities/Compiler/Options/OptionKey.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Threading;
+
+namespace Analyzer.Utilities
+{
+    internal readonly struct OptionKey : IEquatable<OptionKey>
+    {
+        private static readonly ConcurrentDictionary<(string ruleId, string optionName), OptionKey> s_keys = new();
+        private static int _lastOrdinal;
```

Comment:
Do you use the `s_` prefix only for readonly static fields?

Reply:
Supposed to be all static fields I think.

Reply:
Fixed